swagger.yml

```diff
@@ -55,6 +55,9 @@ definitions:
         $ref: '#/definitions/konto'
       kundenbetreuer:
         $ref: '#/definitions/kundenbetreuer'
+      vertriebskanal:
+        description: Zur Aussteuerung der Dokumentegenerierung.
+        $ref: '#/definitions/vertriebskanal'
```

Comment:
@larsp79  Darlegung finde ich überzeugend.

Seinen Vorschlag verstehe ich so, dass man die Definition des enums drin läßt - sie aber nur in der Beschreibung referenziert und als Typ string verwendet.

Reply:
Ja okay, das find ich auch gut.